game.js

```diff
@@ -53,13 +129,15 @@ document.addEventListener("click", (event) => {
 
     game.xTurn = !game.xTurn;
 
+//3X3 game    
```

Comment:
you don't need this comment (you have the if(board3x3) condition which clarify this point)